### ISSUE TITLE
`dispatch` missing from Typescript typings

### ISSUE THREAD

**Issue by scflode** [hybridsjs/hybrids#88]
I just found out that the `dispatch` function seems to be missing from the `index.d.ts` file.

It should be something like this (maybe `options` can be more explicit):

```
  function dispatch(host: Element, eventType: string, options?: any): Boolean
```

I looked into the file and wasn't sure where to put it so maybe one of the maintainers can make a PR.

Thanks for this nice library!

**Comment by smalluban** [comment]
Yes indeed - I'm on it already, I discovered that yesterday. Also `parent` and `children` declarations need slight modification (additional generic type, for now, they are not compatible with `Hybrids` requirements).

I should have ready PR soon.

**Comment by scflode** [comment]
Wow. That was a quick response. Thanks!

**Comment by smalluban** [comment]
The fix is already on master - https://github.com/hybridsjs/hybrids/commit/2ab41bc0b20c1c259341d1eed4f75e0a86169c79 - I will wait with the release with pending PRs - so it should be public tomorrow.

**Comment by scflode** [comment]
Thank you very much!

**Comment by scflode** [comment]
Maybe I'm wrong but I just saw that `options` is not optional:

```
function dispatch(host: HTMLElement, eventType: string, options: CustomEventInit): boolean;
```

Shouldn't it rather be:

```
function dispatch(host: HTMLElement, eventType: string, options?: CustomEventInit): boolean;
```
?

**Comment by smalluban** [comment]
Yep, my mistake, thanks for letting me know - I amended the commit, so here it is - https://github.com/hybridsjs/hybrids/commit/5797efcef839c63d491e97d4e68c0fb56f1101db

**Comment by smalluban** [comment]
Published in `v4.1.1`.
